lib/signature.coffee

```diff
@@ -93,10 +95,13 @@ module.exports = class Signature
 				if parameter.isOptional() and _.isEmpty(value)
 					return result
 
-				result[parameter.getValue()] = value
+				result[parameterValue] = value
 				return result
 
 			if not parameter.isWord() and word?
-				result[parameter.getValue()] = _.parseInt(word) or word
+				if /^\d+$/.test(word)
```

Comment:
Spaces support is covered by surrounding the string with quotes (remember the regex snippet you helped me with), as otherwise it'd be ambiguos whether a certain word belongs to the current parameter or not.

About comma-separated numbers, it's a good idea and I'll note it down.